searcher/src/main/java/ru/otus/searcher/applicationRunner/CountryLoadingService.java

```diff
@@ -0,0 +1,46 @@
+package ru.otus.searcher.applicationRunner;
+
+import dto.CountryDto;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.otus.searcher.controllers.CountrySearchController;
+import ru.otus.searcher.entity.Country;
+import ru.otus.searcher.repository.CountryRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+@Slf4j
+public class CountryLoadingService implements DataLoader{
+    private final CountrySearchController countrySearchController;
+    private final CountryRepository countryRepository;
+
+    @Override
+    public void load() {
+        List<CountryDto> countries = countrySearchController.getCountries();
+        List<Country> dtoToEntity = convert(countries);
+        List<Country> countriesSavedToDb = countryRepository.saveAll(dtoToEntity);
+
+        log.info("countries counter got from travelPayout: {}", countries.size());
```

Comment:
better to log right after getting the data in case next instruction fails - that leads to missing logs